scss/_patterns_icons.scss

```diff
@@ -104,7 +104,7 @@ $social-icon-size: map-get($icon-sizes, social);
 }
 
 @mixin vf-icon-delete($color) {
-  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' height='16' width='16'%3E%3Cg color='%23000'%3E%3Cpath opacity='.11' fill='none' d='M0 0h16v16H0z'/%3E%3Cpath opacity='.11' fill='none' d='M0 0h16v16H0z'/%3E%3Cpath opacity='.11' fill='none' d='M0 0h16v16H0z'/%3E%3Cpath style='text-decoration-color:%23000;isolation:auto;mix-blend-mode:normal;block-progression:tb;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-transform:none' d='M2 4v1h2V4H2zm11 0v1h2V4h-2zM2 6v8.5c0 .83.68 1.5 1.5 1.5h10c.82 0 1.5-.67 1.5-1.5V6h-2v7.5c0 .3-.21.5-.5.5h-8a.49.49 0 0 1-.5-.5V6H2z' fill='#{vf-url-friendly-color($color)}'/%3E%3Cpath d='M6 0v3h1V1h3v2h1V0H6zM5 6h1v6H5zM8 6h1v6H8zM11 6h1v6h-1z' fill='#{vf-url-friendly-color($color)}'/%3E%3Cpath style='text-decoration-color:%23000;isolation:auto;mix-blend-mode:normal;block-progression:tb;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-transform:none' d='M3.5 2C2.68 2 2 2.68 2 3.5V5h13V3.5c0-.82-.68-1.5-1.5-1.5h-10zM2 6v8h2V6H2zm11 0v8h2V6h-2z' fill='#{vf-url-friendly-color($color)}'/%3E%3C/g%3E%3C/svg%3E");
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='16'%3E%3Cg fill='none'%3E%3Cpath d='M-1 0h16v16H-1z'/%3E%3Cpath fill='gray' d='M1.5 4.998h-1v-1.5C.5 2.678 1.178 2 2 2h2.5V0h5v2H12c.822 0 1.5.677 1.5 1.499v1.5H.5v-1h2v1h-1zm10 8.503V5.998h2V14.5c0 .822-.678 1.499-1.5 1.499H2c-.822 0-1.5-.677-1.5-1.5V5.999h2V13.5c0 .285.214.5.5.5h8c.286 0 .5-.215.5-.5zm-3-11.502V1h-3v1h3zm-5 3.999h1v5.998h-1V5.998zm3 0h1v5.998h-1V5.998zm3 0h1v5.998h-1V5.998z'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
Great that we not only got icon fixed but also cleaned up!

Reply:
Thanks @bartaz developer in the making 😉